README.rst

```diff
@@ -92,26 +83,149 @@ Or, on a custom port and address::
 Any changes made to Python will be detected and rebuilt transparently as
 long as the development server is running.
 
+Not the live site
+-----------------
+
+This project is not intended to serve the license and deed pages directly.
+Though if it's deployed on a public server it could do that, performance
+would probably not be acceptable.
+
+Instead, a command line tool can be used to save all the rendered HTML
+pages for licenses and deeds as files. Then those files are used as part
+of the real creativecommons.org site, just served as static files.
+See details farther down.
+
+Data
+----
+
+The license data is stored as follows.
+
+The license metadata is in a database. The metadata tracks which licenses exist, their translations,
+their ports, and their characteristics like what they permit, require, and prohibit.
+
+The metadata can be downloaded by visiting `/licenses/metadata.yaml`.
+
+There are two main models (that's Django terminology for tables).
+
+A License can be identified by a license code (e.g. BY, BY-NC-SA) which
+is a proxy for the complete set of permissions, requirements, and
+prohibitions; a version number (e.g. 4.0, 3.0), and an optional jurisdiction
+for ports. So we might refer to the license "BY 3.0 Armenia" which would be
+the 3.0 version of the BY license terms as ported to the Armenia jurisdiction.
+
+A License can exist in multiple languages or translations. Each one, including
+English, is represented by a LegalCode record. A LegalCode is identified by
+a license and a language, e.g. we might refer to the "BY 3.0 Armenia in Armenian"
+legalcode record.
+
+For some licenses, the text is in gettext files (.po and .mo) in the cc-licenses-data
+repository. For others, the text is stored in the database, directly in
+LegalCode records.
```

Comment:
I think we should mention which license's text is stored in cc_licenses_data and which is stored in the database.